aws/resource_aws_lb_cookie_stickiness_policy_test.go

```diff
@@ -182,7 +183,7 @@ resource "aws_lb_cookie_stickiness_policy" "foo" {
   name                     = "foo-policy"
   load_balancer            = aws_elb.lb.id
   lb_port                  = 80
-  cookie_expiration_period = 300
```

Comment:
Typically it is preferable to create new test cases rather than adjust existing tests to ensure we don't miss any regressions, but in this case it is probably okay. Just note that the comment on line 167 is now incorrect and we should probably be verifying these state values with `resource.TestCheckResourceAttr()` in the test functions.

Reply:
I figured it was probably ok in this case since it was still testing the update functionality. I got the comment fixed and also added checks for the `cookie_expiration_period`.